implementations/danubetech/src/main/java/ldsignatures.test.jws/JWSTestSuite.java

```diff
@@ -87,34 +43,83 @@ static void create(String input, String key, String outputFilename, String proof
         jsonWebSignature2020LdSigner.setChallenge("123");
         jsonWebSignature2020LdSigner.sign(jsonLDObject, true, false);
         String output = jsonLDObject.toJson(true);
-        writeFile(outputFilename, output);
+        FileUtil.writeFile(outputFilename, output);
+    }
+
+    static void credentialCreate(String input, URI verificationMethod, ByteSigner byteSigner, String outputFilename) throws JsonLDException, GeneralSecurityException, IOException {
+        create(input, verificationMethod, byteSigner, outputFilename, LDSecurityKeywords.JSONLD_TERM_ASSERTIONMETHOD);
+    }
+
+    static void presentationCreate(String input, URI verificationMethod, ByteSigner byteSigner, String outputFilename) throws JsonLDException, GeneralSecurityException, IOException {
+        create(input, verificationMethod, byteSigner, outputFilename, LDSecurityKeywords.JSONLD_TERM_AUTHENTICATION);
+    }
+
+    static void createJwt(JwtObject jwtObject, URI verificationMethod, ByteSigner byteSigner, String outputFilename) throws JOSEException, IOException {
+        String jwtString;
+        if (JWSAlgorithm.EdDSA.equals(byteSigner.getAlgorithm()))
+            jwtString = jwtObject.sign_Ed25519_EdDSA(byteSigner, verificationMethod.toString(), false);
+        else if (JWSAlgorithm.ES256K.equals(byteSigner.getAlgorithm()))
+            jwtString = jwtObject.sign_secp256k1_ES256K(byteSigner, verificationMethod.toString(), false);
+        else if (JWSAlgorithm.PS256.equals(byteSigner.getAlgorithm()))
+            jwtString = jwtObject.sign_RSA_RS256(byteSigner, verificationMethod.toString(), false);
```

Comment:
Should this be PS256?
```suggestion
            jwtString = jwtObject.sign_RSA_PS256(byteSigner, verificationMethod.toString(), false);
```